face_detection.mjs

```diff
@@ -0,0 +1,77 @@
+import { resize } from "./preprocess.mjs";
+import { getOrt } from "./onnxruntime.mjs";
+import { tensorToMatrix } from "./inference_utils.mjs";
+
+export class GuruFaceDetector {
+  /**
+   * Initializes the GuruFaceDetector for face detection using a pre-trained ONNX model.
+   *
+   * The constructor expects a 'guruModel' object which contains the ONNX model session
+   * specifically trained for face detection tasks. This model is utilized to perform
+   * inference on the input images. Additionally, an ONNX Runtime (ORT) instance is
+   * retrieved and stored for further use in the model inference process.
+   *
+   * @param {Object} guruModel - An object containing the ONNX model session.
+   *                             This is typically loaded from an external ONNX model file.
+   */
+  constructor(guruModel) {
+    this.model = guruModel.session;
```

Comment:
For all of the other classes that wrap an ONNX model (e.g., [YOLOXDetector](https://github.com/Formguru/guru-js/blob/ecaa126f57d2a466c156c43a59ff5c4fad6c0c1c/object_detection.mjs#L21), [GuruPoseEstimator](https://github.com/Formguru/guru-js/blob/ecaa126f57d2a466c156c43a59ff5c4fad6c0c1c/pose_estimation.mjs#L13)), the constructor takes an InferenceSession (or an `Array[InferenceSession]` in the case of partitioned models, but that doesn't apply here).

Not sure if that was the right decision but we should probably keep it consistent here.

Reply:
@astahlman is the latest commit what you're expecting? From the schema, I'd instantiate the face detector by:

`this.faceDetector = new UltrafaceFaceDetector(frame.guruModels.faceDetectorModel().session);`